unit-tests-jest/numbers.ts

```diff
@@ -9,18 +9,23 @@ export function evenNumbers(numbers: number[]): number[] {
  * Returns a number formatted in dollars and cents.
  */
 export function toDollars(amount: number): string {
-  return `$${amount}.00`;
+  if (amount % 1 !== 0) {
```

Comment:
It works fine without this check. `amount.toFixed(2)` will return, say, `3.00` for integers.